test/groovy/TransportRequestUploadFileTest.groovy

```diff
@@ -192,11 +192,15 @@ public class TransportRequestUploadFileTest extends BasePiperTest {
     }
 
     @Test
-    public void uploadFileToTransportRequestSOLMANSuccessTest() {
+    public void uploadFileToTransportRequestSOLMANTransportRequestIdViaParametersSuccessTest() {
```

Comment:
How does the changes in that file fit into the docu scope of this PR? Groundfog, I suppose...